src/components/AppHeader.jsx

```diff
@@ -287,8 +290,7 @@ class AppHeader extends Component {
                 <Menu.Item className='orcidInfo'>{orcid_info()}</Menu.Item>
 
                 <Menu.Item className='sessionInfo'>
-                    Your session will be deleted in{' '}
-                    {moment.duration(timeout).humanize()} &nbsp;
+                    Your session will be deleted {timeout} &nbsp;
```

Comment:
This is still "hunamised", but uses built in browser APIs rather than the old moment.js solution. This means I can remove the moment.js dependency entirely since this was the only use.